docs/reference/operator_parameters.md

```diff
@@ -365,6 +365,9 @@ key.
   role name to grant to team members created from the Teams API. The default is
   `admin`, that role is created by Spilo as a `NOLOGIN` role.
 
+* **add_admin_flag_to_robot_roles**
```

Comment:
enable_admin_role_for_users

Could be good enough, the others are "admin/superusers" and "team members" so maybe "users" is good enough.